File: lib/payment_result.dart
class PaymentResult {
  final String status;
  final String terminalID;
  final String stan;
  final String rrn;
  final String responseCode;
  final String customerCardNO;
  final String transactionDateTime;
  final String description;
  const PaymentResult({
    required this.status,
    required this.terminalID,
    required this.stan,
    required this.rrn,
    required this.responseCode,
    required this.customerCardNO,
    required this.transactionDateTime,
    required this.description,
  });

  static PaymentResult fromjson(Map json) {
    return PaymentResult(
      status: json['status'],
      terminalID: json['terminalID'],
      stan: json['stan'],
      rrn: json['rrn'],
      responseCode: json['responseCode'],
      customerCardNO: json['customerCardNO'],
      transactionDateTime: json['transactionDateTime'],
      description: json['description'],
    );
  }
}

File: android/app/src/main/kotlin/com/example/pos_flutter/models/PaymentResult.java
package com.example.pos_flutter.models;

public class PaymentResult {
    private String Status;
    private String TerminalID;
    private String STAN;
    private String RRN;
    private String ResponseCode;
    private String CustomerCardNO;
    private String TransactionDateTime;
    private String Description;

    public String getStatus() {
        return Status;
    }

    public void setStatus(String status) {
        Status = status;
    }

    public String getTerminalID() {
        return TerminalID;
    }

    public void setTerminalID(String terminalID) {
        TerminalID = terminalID;
    }

    public String getSTAN() {
        return STAN;
    }

    public void setSTAN(String STAN) {
        this.STAN = STAN;
    }

    public String getRRN() {
        return RRN;
    }

    public void setRRN(String RRN) {
        this.RRN = RRN;
    }

    public String getResponseCode() {
        return ResponseCode;
    }

    public void setResponseCode(String responseCode) {
        ResponseCode = responseCode;
    }

    public String getCustomerCardNO() {
        return CustomerCardNO;
    }

    public void setCustomerCardNO(String customerCardNO) {
        CustomerCardNO = customerCardNO;
    }

    public String getTransactionDateTime() {
        return TransactionDateTime;
    }

    public void setTransactionDateTime(String transactionDateTime) {
        TransactionDateTime = transactionDateTime;
    }

    public String getDescription() {
        return Description;
    }

    public void setDescription(String description) {
        Description = description;
    }
}
